malduck/extractor/extractor.py

```diff
@@ -40,20 +47,20 @@ def __new__(cls, name, bases, attrs):
         return klass
 
 
-class ExtractorMethod(object):
+class ExtractorMethod:
     """
     Represents registered extractor method
     """
 
-    def __init__(self, method):
+    def __init__(self, method: Callable[..., Config]) -> None:
```

Comment:
Actually, it's not really correct. `ExtractorMethod` can return None and bool when we want to indicate a match without providing actual configuration (e.g. flush weak matches)
```
    def __init__(self, method: Callable[..., Union[Config, bool, None]]) -> None:
```

Currently it shows additional warnings with `--strict-equality`

```
malduck/extractor/extractor.py:69: error: Non-overlapping identity check (left operand type: "Dict[str, Any]", right operand type: "Literal[True]")
```